.github/workflows/build.yml

```diff
@@ -19,8 +19,13 @@ jobs:
       matrix:
         os:
           - windows-2019
+          - ubuntu-latest
         node:
           - 10
+          - 12
+          - 14
+          - 16
+          - 18
```

Comment:
```suggestion
          - 18
          - 20
```

Reply:
We have to work our way up to Node 18/20. node-tree-sitter has [superstring]() in its devDependencies. superstring doesn't support Node 17+, so if you try to run `npm install` in node-tree-sitter on Node 18 or 20, it will fail.